### PR TITLE
Added CLI and SMI-S access methods. Updated registration forms

### LINKED ISSUE #525: [Delfin UI] Add new storage devices to the list of supported devices and allow user to register new devices.
**Issue/Feature Description:**
Delfin now supports the following new devices. Allow the user to register these devices from the SODA Dashboard.

- [x] VPLEX driver registration
- [x] VNX driver registration
- [ ] IBM DS8000 driver
- [ ] EMC PowerMax
- [ ] EMC ExtremeIO
- [ ] IBM FlashSystem
- [ ] NetApp FA

**Why this issue to fixed / feature is needed(give scenarios or use cases):**

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### LINKED ISSUE #526: [Delfin UI] Add new methods of access for storage devices: CLI and SMI-S
**Issue/Feature Description:**
Allow the user to register storage devices with additional access methods:

- [x] CLI
- [x] SMI-S

**Why this issue to fixed / feature is needed(give scenarios or use cases):**

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
/kind enhancement

**What this PR does / why we need it**:
This PR adds the CLI and SMI-S access mechanism support for storage device registration from SODA Dashboard using SODA Delfin.
This PR also adds support for the Dell EMC VNX driver.

**Which issue(s) this PR fixes**:
Fixes  #526 

**Test Report Added?**:
/kind TESTED

**Test Report**:
access-1
access-2
vnx-3
vnx-4

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/shared/utils/consts.ts [hunk 1]
```diff
@@ -188,6 +190,19 @@ export const Consts = {
                         name: 'unity',
                         rest: true,
                         ssh: false,
+                        cli: false,
+                        smis: false,
+                        extra: false
+                    }
+                },
+                {
+                    label: "VNX",
```
Comment (NajmudheenCT) [inline src/app/shared/utils/consts.ts:199]: could you please hold the VNX dictionary entry till we merge the vnx driver in delfin

Comment (anvithks) [reply src/app/shared/utils/consts.ts]: Done. removed the entry for now

**Review by kumarashit — APPROVED** [review]
LGTM

**Comment by joseph-v** [comment]
LGTM

**Review by joseph-v — APPROVED** [review]

**Review by NajmudheenCT — APPROVED** [review]
LGTM

**Pull request merged**